configs/cameras/tptek_4mp

```diff
@@ -0,0 +1,18 @@
+# NAME: tptek-4mp
+# MODULE: t31x_gc4653_eth_rtl8731
+
+BR2_PACKAGE_WIFI=y
```

Comment:
Please reorder parameters alphabetically and drop empty lines. That saves a ton of time when searching and comparing things through dozens of profiles.